.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Are comments supported here? Usually don't exist in json afaik. 🐭

Reply:
VSCode configuing `.json` files (`launch.json`, `settings.json`, etc) actually use `jsonc`, which is basically JSON that allows single-line comments and trailing commas to make configuration via JSON a little less maddening.